### PR TITLE
updating

### PR DESCRIPTION
**Pull request by jasmin596**
git cheat Sheet updated

### CODE REVIEW

File: docs/tools/git-cheat-sheet.md [hunk 1]
```diff
@@ -3,25 +3,176 @@ id: git
...
 ---
+**# git && commands**
 
-# git && commands
+# **git**
 
-## Git Commands
-1. git config
+Git is a distributed version control system (DVCS). "Distributed" means that all developers within a team have a complete version of the project. A version control system is simply software that lets you effectively manage application versions.
+ you'll be able to do the following:
```
Comment (rabbani8) [inline docs/tools/git-cheat-sheet.md:11]: make it proper description

File: docs/tools/git-cheat-sheet.md [hunk 2]
```diff
@@ -3,25 +3,176 @@ id: git
...
+ *  Compare and analyze code
+ *  Merge code from_ different computers and different      team members.
 
+# Git Commands
 
+## Configuring Git
 
+**git config**
 
+``` Usage: git config –global user.name “[name]”
```
Comment (rabbani8) [inline docs/tools/git-cheat-sheet.md:25]: why we are  using this give short description there

File: docs/tools/git-cheat-sheet.md [hunk 3]
```diff
@@ -3,25 +3,176 @@ id: git
...
+    Usage: git config –global user.email “[email address]” 
+    Usage: git config --global color.ui true
+    Usage: git config --list
+```
+![iloveA-i](assets/image.png)
```
Comment (rabbani8) [inline docs/tools/git-cheat-sheet.md:30]: why name of image is iloveA-i ?

File: docs/tools/git-cheat-sheet.md [hunk 4]
```diff
@@ -3,25 +3,176 @@ id: git
...
+
+![iloveA-i](assets/status.png)
+
+**git branch**
+```
+ Usage: git branch
+```
+The "branch" command helps you create and list branches. It's the go-to command when it comes to managing any aspect of your branches. No matter if in your local repository or on your remotes.
+
+![iloveA-i](assets/branch.png)
```
Comment (rabbani8) [inline docs/tools/git-cheat-sheet.md:67]: iloveA-i   why this name ?

File: docs/tools/git-cheat-sheet.md [hunk 5]
```diff
@@ -3,25 +3,176 @@ id: git
...
+git merge [branch name] | Merge a branch into the active branch
+git merge [source branch] [target branch] |Merge a branch into a target branch
+git stash |Stash changes in a dirty working directory
+git stash clear  |Remove all stashed entries 
+
+
+
+# Sharing & Updating Projects
+
+Command |Description
```
Comment (rabbani8) [inline docs/tools/git-cheat-sheet.md:159]: good, keep it up

**Comment by jasmin596** [comment]
I have done changes

**Comment by jasmin596** [comment]
Tqq

**Pull request merged**